process-engine/bpmn-studio
# ✨:sparkles: Check Unique Ids

## What did you change?

This PR adds a feature to check whether an ID or Process-ID is unique.
If not, you cannot safe and a warning appears.

Fixes #163 

## How can others test the changes?

1. Checkout the branch.
2. npm install
3. au run --watch
4. Open a diagram, open an element and type an existing Id
5. A warning should appear inside the ID field.

Same for Process ID.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
